projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -16,12 +17,23 @@ import SupportSection from '../support-section';
 import ConnectionScreen from './../connection-screen';
 import Header from './../header';
 import InfoSection from './../info-section';
+import InstagramNotice from './../instagram-notice';
 import AdminPageHeader from './header';
 import './styles.module.scss';
 
 const Admin = () => {
 	const { isUserConnected, isRegistered } = useConnection();
 	const showConnectionCard = ! isRegistered || ! isUserConnected;
+	const [ manuallyShowPricingPage, setManuallyShowPricingPage ] = useState( false );
+
+	const onUpgradeToggle = useCallback(
+		() => setManuallyShowPricingPage( true ),
+		[ setManuallyShowPricingPage ]
+	);
+	const onPricingPageDismiss = useCallback(
+		() => setManuallyShowPricingPage( false ),
+		[ setManuallyShowPricingPage ]
+	);
```

Comment:
The reference of the state setter from `useState` never changes, so those can be omitted from the dependencies.

Reply:
Ah good to know, fixed!